slasher/Cargo.toml

```diff
@@ -17,7 +17,8 @@ eth1_api = { workspace = true }
 features = { workspace = true }
 futures = { workspace = true }
 helper_functions = { workspace = true }
-log = { workspace = true }
+tracing = { workspace = true }
```

Comment:
[**chore**](https://conventionalcomments.org/): Keep dependencies in alphabetical order